screens/CheckoutScreen.js

```diff
@@ -181,61 +188,113 @@ export default class CheckoutScreen extends React.Component {
+  getIndexOfFirstProductAtLetter = letter => {
+    let prodList = this.state.products.filter(product => product.name.charAt(0) === letter);
+    let nextLetter = letter;
+    while (prodList.length === 0) {
+      nextLetter = String.fromCharCode(nextLetter.charCodeAt(0) + 1);
+      prodList = this.state.products.filter(product => product.name.charAt(0) === nextLetter);
+    }
+    return this.state.products.indexOf(prodList[0]);
+  };
+
+  // Takes in strings tab label (i.e. "A-K") and starting letter (i.e. "A") and returns a
+  // tab for the bottom alphabetical scroll bar.
+  alphabeticalScrollTab = (label, letter) => {
+    return (
+      <TabContainer
+        onPress={() =>
+          this._scrollView.scrollTo({
+            y: Math.floor(this.getIndexOfFirstProductAtLetter(letter) / 5) * 160
+          })
+        }>
+        <Title>{label}</Title>
+      </TabContainer>
+    );
+  };
+
   render() {
     if (this.state.isLoading) {
       return null;
     }
 
     const { cart, customer, totalBalance } = this.state;
     const totalSale = totalBalance > 0 ? totalBalance : 0;
+    const pointsEarned = this.getPointsEarned();
+    const subtotal = totalBalance + this.state.rewardsApplied * rewardDollarValue;
 
     return (
-      <View>
+      <View style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-around', flex: 1 }}>
         <TopBar>
           <BackButton navigation={this.props.navigation} light={false} style={{ marginTop: 3, marginLeft: 24 }} />
           <Title> {'Customer: '.concat(customer.name)} </Title>
           {/* Duplicate, invisible element to have left-aligned BackButton */}
           <BackButton navigation={this.props.navigation} light={false} style={{ opacity: 0.0, disabled: true }} />
         </TopBar>
-        <View style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-around' }}>
+        <View style={{ display: 'flex', flexDirection: 'row', flex: 1 }}>
           {/* Display products */}
-          <ProductsContainer>
-            {Object.entries(cart).map(([id, product]) => (
-              <QuantityModal key={id} product={product} isLineItem={false} callback={this.updateQuantityCallback} />
-            ))}
-          </ProductsContainer>
+          <View style={{ display: 'flex', flex: 5, flexDirection: 'column' }}>
+            <ProductsContainer
+              ref={scrollView => {
+                this._scrollView = scrollView;
+              }}>
+              {Object.entries(cart).map(([id, product]) => (
+                <QuantityModal key={id} product={product} isLineItem={false} callback={this.updateQuantityCallback} />
+              ))}
+            </ProductsContainer>
+            <BottomBar style={{ display: 'flex', flexDirection: 'row' }}>
+              {this.alphabeticalScrollTab('A-K', 'A')}
+              {this.alphabeticalScrollTab('L-S', 'L')}
+              {this.alphabeticalScrollTab('T-Z', 'T')}
```

Comment:
Can we add spaces between the hyphens like 'A - K' so it looks less cramped? (Ace approved)